hs_model_program/models.py

```diff
@@ -110,6 +110,10 @@ def can_delete(self, request):
     def can_view(self, request):
         return AbstractResource.can_view(self, request)
 
+    @classmethod
```

Comment:
This change allows files to be uploaded after the resource has already been created.  Without this, an error prevents file upload.